met-web/src/routes/AuthenticatedRoutes.tsx

```diff
@@ -3,17 +3,12 @@ import { Route, Routes } from 'react-router-dom';
 import NotFound from './NotFound';
 import LandingPage from './LandingPage/LandingPage';
 import { ThemeProvider } from '@mui/system';
-import { BaseTheme, PublicTheme } from '../styles/Theme';
+import { BaseTheme} from '../styles/Theme';
 import UserService from '../services/UserServices';
 import View from '../components/Form/View';
-import CreateEngagementForm from '../components/engagement/CreateEngagementForm';
 import Engagement from '../components/engagement';
 
 const AuthenticatedRoutes = () => {
-    let adminRole = UserService.hasAdminRole();
```

Comment:
I understand this creates a lint issue and it is annoying. But maybe it is useful to us to know the future use here? Is it okay to add it again? @saravanpa-aot  thoughts here? Idea is theme={adminRole ? BaseTheme : PublicTheme}.